### PR TITLE
Fix path in plugin boostrapping

### LINKED ISSUE #68: Fix path in plugin boostrapping
Need to use JPATH_PLUGINS in https://github.com/joomlatools/joomlatools-framework/blob/master/code/plugins/system/joomlatools/joomlatools.php#L165 to make joomlatools-framework work in joomlatools-platform.

### PR DESCRIPTION
**Pull request by allanpilarca**
close #68

### CODE REVIEW

File: code/plugins/system/joomlatools/joomlatools.php [hunk 1]
```diff
@@ -162,7 +167,7 @@ public function bootstrap()
              */
             $loader->registerLocator(new ComKoowaClassLocatorPlugin(array(
                 'namespaces' => array(
-                    '\\'     => JPATH_ROOT.'/plugins',
+                    '\\'     => JOOMLATOOLS_PLATFORM ? JPATH_PLUGINS : JPATH_ROOT.'/plugins',
```
Comment (johanjanssens) [inline code/plugins/system/joomlatools/joomlatools.php]: @allanpilarca Wouldn't it be sufficient here to do a defined('JOOMLATOOLS_PLATFORM') check?  If this define exists we can assume we are in platform context.

Comment (allanpilarca) [inline code/plugins/system/joomlatools/joomlatools.php]: @johanjanssens  Yes, that is simpler way to check. No need to check it in the composer.json file. Applied your suggestion. Thanks :)

**Comment by johanjanssens** [comment]
@ercanozkaya Can you double check please and merge?

**Comment by ercanozkaya** [comment]
@johanjanssens @allanpilarca Why don't we directly use `JPATH_PLUGINS`? Then there would be no need to check for platform or not?

**Comment by johanjanssens** [comment]
@ercanozkaya True. @allanpilarca In platform JPATH_PLUGINS is set too the right folder, see:  https://github.com/joomlatools/joomlatools-platform/blob/4401b3685bb9fd4b36b1477b46b14ba6a1bba5fe/app/defines.php

**Pull request merged**